tests/meta-mender-ci/recipes-testing/read-only-rootfs/mender_%.bbappend

```diff
@@ -0,0 +1,12 @@
+
+IMAGE_FEATURES_append = " read-only-rootfs"
+
+FILES_${PN} += "/data/etc/mender"
+# Run after all mender recipes are finished.
+do_install_append() {
+    mkdir -p ${D}/data/etc/mender
+    mv ${D}/etc/mender/mender.conf ${D}/data/etc/mender
+    mv ${D}/etc/mender/server.crt ${D}/data/etc/mender
+    ln -s /data/etc/mender/mender.conf ${D}/etc/mender/mender.conf
```

Comment:
@oleorhagen: I have hard proof now. This link exists in the Docker client image, which means there is definitely something wrong in the build, and the link somehow survives the acceptance test stage and gets into the integration tests. I still have not found out why though.

Reply:
Btw, even stranger is that the link exists, but its `/data` counterpart does not, so only half of the changes survived. Could it be that the `.mender` artifact survives with the link change (what is on the rootfs), and the `.sdimg` does not (what is in `/data`)? Mysterious...